types/eslint-utils/referenceTracker.d.ts

```diff
@@ -0,0 +1,62 @@
+import { Scope } from 'eslint';
+import { Node } from 'estree';
+
+export interface ReferenceTrackerOptions {
+    /**
+     * The variable names for Global Object.
+     */
+    globalObjectNames?: string[];
+
+    /**
+     * The mode to determine the ImportDeclaration's behavior for CJS modules.
+     */
+    mode?: 'legacy' | 'strict';
+}
+
+export interface TraceMap {
+    [i: string]: TraceMapObject;
+}
+
+export interface TraceMapObject {
+    [i: string]: TraceMapObject;
+    [CALL]?: boolean;
+    [CONSTRUCT]?: boolean;
+    [ESM]?: boolean;
+    [READ]?: boolean;
+}
+
+export const CALL: unique symbol;
+export const CONSTRUCT: unique symbol;
+export const ESM: unique symbol;
+export const READ: unique symbol;
+
+export interface TrackedReferences {
+    info: any;
```

Comment:
https://eslint-utils.mysticatea.dev/api/scope-utils.html#tracker-iteratecjsreferences

```suggestion
    entry: unknown;
```